app/code/pagetype/HomePage.php

```diff
@@ -3,10 +3,21 @@
 namespace App\PageType;
 
 use Page;
-use SilverStripe\Forms\TextField;
 use SilverStripe\Assets\Image;
+use SilverStripe\Forms\TextField;
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Forms\TreeDropdownField;
 use SilverStripe\AssetAdmin\Forms\UploadField;
 
+/**
+ * The HomePage of the website
+ *
+ * @category Page
```

Comment:
Awesome stuff adding all these attributes but this can be something that we can ignore on a class level as this is documented on our read me file and the `author` of this class will also change from time to time. So am happy for you to remove these :) 

We only usually add a docblock if there is complex logic in the class :)